src/Discord.Net.Commands/CommandService.cs

```diff
@@ -83,20 +83,21 @@ public CommandService(CommandServiceConfig config)
                 _moduleLock.Release();
             }
         }
-        public async Task<ModuleInfo> AddModuleAsync<T>()
+        public async Task<ModuleInfo> AddModuleAsync<T>() => await AddModuleAsync(typeof(T));
```

Comment:
Remove `async` and `await` here, there's no need to use an async context as we're just redirecting to the other method.